make.js

```diff
@@ -307,11 +309,22 @@ target.dist = function() {
     GENERIC_DIR + 'build/pdf.js',
     GENERIC_DIR + 'build/pdf.worker.js',
     SINGLE_FILE_DIR + 'build/pdf.combined.js',
+    MINIFIED_DIR + 'build/pdf.min.js',
+    MINIFIED_DIR + 'build/pdf.worker.min.js',
+    MINIFIED_DIR + 'build/pdf.combined.min.js'
   ], DIST_DIR + 'build/');
 
   mkdir('-p', DIST_DIR + 'web/');
   cp('-R', [
     COMPONENTS_DIR + '*',
+    GENERIC_DIR + 'web/viewer.js',
```

Comment:
Please note that the *only* files that should be present in the `dist` are the ones listed here: https://github.com/mozilla/pdfjs-dist/tree/master/web.
Providing minified versions of those files should be OK, but `viewer.{js, css}` should *not* be included!